### ISSUE TITLE
Attested image already whitelisted from container registry anyway

### ISSUE THREAD

**Issue by mjlshen** [GoogleCloudPlatform/gke-binary-auth-demo#25]
First off, thanks for the very informative demo!

This may be a bit pedantic, but in the instructions/README under the subtitle "**Denying Images Except From Whitelisted Container Registries**," exempting images from this container registry path:

`echo "gcr.io/${PROJECT_ID}/nginx*"`

would also exempt the image that is attested at the end of the lab right (that is, regardless of whether or not the attestation was successful)? I think a step or sentence should be added in to remove that image path from the whitelist a "true" test of the image attestation. Happy to give it a shot, but since this is very new to me I want to make sure I'm not misunderstanding something.

**Comment by Fab-T** [comment]
I agree with you, I confirm that if you don't remove your private repo from the whitelist, no matter the attestation, it loads the pod with the image.
